React/Recipes/Characters encoding issue in static pages in NextJS.md

```diff
@@ -0,0 +1,53 @@
+## Motivation
+
+Although this is not an issue with NextJS but React (check more details [here](https://github.com/facebook/react/issues/13838)),
+you can encounter this if you have the static pages, and those static pages have some HTML entities in the page content -
+e.g. ampersand (`&`) in query parameters.
+
+The static pages will have the HTML entities being encoded, which will result in incorrect content. Probably it will be
+decoded correctly on the client, but this might be an issue for the crawlers for your static content.
```

Comment:
```suggestion
These HTML entities will not be encoded by default, which will result in incorrect content. This will pose an issue mostly for crawlers.
```